californium-core/src/main/java/org/eclipse/californium/core/network/RandomTokenGenerator.java

```diff
@@ -63,7 +63,7 @@ public RandomTokenGenerator(final NetworkConfig networkConfig) {
 		// trigger self-seeding of the PRNG, may "take a while"
 		this.rng.nextInt(10);
 		this.tokenSize = networkConfig.getInt(Keys.TOKEN_SIZE_LIMIT, DEFAULT_TOKEN_LENGTH);
-		LOGGER.info("using tokens of {} bytes in length", this.tokenSize);
+		LOGGER.trace("using tokens of {} bytes in length", this.tokenSize);
```

Comment:
The intention of info is to log the effective configuration on start. So I would keep info.